### PR TITLE
Bevy 0.14

### PR DESCRIPTION
**Pull request by csandven**

### CODE REVIEW

**Comment by cyclopanic** [comment]
ss
not sure why the tilemap example seems to not render properly

**Review by Rezan7CC — CHANGES_REQUESTED** [review]
I like the formatting changes but would it be possible to make those separately from the Bevy upgrade?

**Comment by zaycev** [comment]
@cyclopanic Not a blocker for the review, but I agree with @Rezan7CC. Do you think it's possible to submit formatting changes as a separate PR?

File: examples/krypta.rs [hunk 1]
```diff
@@ -20,11 +23,10 @@ pub struct MouseLight;
 #[derive(Component)]
 pub struct Movable;
 
-fn main()
-{
+fn main() {
     // Basic setup.
     App::new()
-        .insert_resource(ClearColor(Color::rgba_u8(0, 0, 0, 0)))
+        .insert_resource(ClearColor(Color::srgba_u8(0, 0, 0, 0)))
```
Comment (zaycev) [inline examples/krypta.rs:29]: Wonder why we should use a different color space?

**Comment by tigerplush** [comment]
Color space has changed according to https://bevyengine.org/learn/migration-guides/0-13-to-0-14/#overhaul-color

File: src/gi/compositing.rs [hunk 2]
```diff
@@ -197,16 +182,16 @@ pub fn setup_post_processing_camera(
...
     *camera_targets = CameraTargets::create(&mut images, &target_sizes);
 
     let material = PostProcessingMaterial::create(&camera_targets, &gi_targets_wrapper);
-    materials.insert(POST_PROCESSING_MATERIAL.clone(), material);
+    materials.insert(POST_PROCESSING_MATERIAL.id(), material);
 
     // This specifies the layer used for the post processing camera, which
     // will be attached to the post processing camera and 2d quad.
-    let layer = RenderLayers::layer((RenderLayers::TOTAL_LAYERS - 1) as u8);
+    let layer = RenderLayers::layer(31);
```
Comment (tigerplush) [inline src/gi/compositing.rs:194]: @csandven what is this magic number?

**Review by tigerplush — CHANGES_REQUESTED** [review]
Please change magic number

**Pull request closed without merging**